src/DependencyInjection/SonataClassificationExtension.php

```diff
@@ -107,131 +108,145 @@ public function registerDoctrineMapping(array $config)
 
         $collector = DoctrineCollector::getInstance();
 
-        $collector->addAssociation($config['class']['category'], 'mapOneToMany', [
-            'fieldName' => 'children',
-            'targetEntity' => $config['class']['category'],
-            'cascade' => [
+        $categoryOnyToManyOptions = OptionsBuilder::create()
```

Comment:
When using the new doctrine extensions, you should check if the `SonataDoctrineBundle` bundle is loaded. Otherwise the defined associations are ignored, because you are doing a static call, not a symfony service call.